kyuubi-server/pom.xml

```diff
@@ -133,6 +133,42 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-avro_${scala.binary.version}</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.hudi</groupId>
+            <artifactId>hudi-spark3_${scala.binary.version}</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.hbase</groupId>
```

Comment:
the test failure is related to this dep? can we exclude it?

Reply:
Hudi needs to depend on hbase.
You mean to exclude it in kyuubi-server module or the whole project?

Reply:
the whole project

Reply:
> Hudi needs to depend on hbase.

just curious, why hudi depend on hbase ?

Reply:
For index

Reply:
ah thanks, it's a good news ..